sentry/Cargo.toml

```diff
@@ -44,6 +44,9 @@ surf-h1 = ["surf_/h1-client", "httpdate"]
 surf = ["surf_/curl-client", "httpdate", "tokio"]
 native-tls = ["reqwest_/default-tls"]
 rustls = ["reqwest_/rustls-tls"]
+ureq = ["ureq_", "httpdate", "tokio"]
```

Comment:
do you happen to know why this needs the `tokio` dependency?  I thought ureq was blocking?  I have the same question for the `surf` dependency as well to be fair, but that's clearly out of scope here :)

Reply:
Because of the `TransportThread` every backend seems to use. `surf` has a clear `requestr.await`, I think that's async driven by `tokio` too? Perhaps you meant `curl` which - just like `ureq` - only has a single `async move { rl }` and is blocking otherwise.

Getting rid of `tokio` is our next stop unless it's simpler/cleaner to implement this directly. Is a `Transport` - or at least the `send_envelope` function - supposed to return quickly or is it fine if we perform a blocking request here?

Reply:
It would be fair to split the `TransportThread` into two sync/async variants for transports that are based on one or the other in the future.

Reply:
oh, the `TrnasportThread` builds an internal Tokio runtime...  that is indeed a bit needless for sync transports and weird for the surf one.  would be nice to fix this sometime indeed.

Reply:
I'll pick this up in a followup PR!

Reply:
https://github.com/getsentry/sentry-rust/pull/422